installer/cli/microk8s.py

```diff
@@ -211,6 +215,36 @@ def kubectl(args) -> int:
         return Linux(args).kubectl()
 
 
+def inspect() -> None:
+    vm_provider_name = "multipass"
+    vm_provider_class = get_provider_for(vm_provider_name)
+    echo = Echo()
+    try:
+        vm_provider_class.ensure_provider()
+        instance = vm_provider_class(echoer=echo)
+        instance.get_instance_info()
+
+        command = ["microk8s.inspect"]
+        output = instance.run(command, hide_output=True)
+        tarball_location = None
+        host_destination = getcwd()
+        if b"Report tarball is at" not in output:
+            echo.error("Report tarball not generated")
+        else:
+            for line in output.split(b"\n"):
+                if line.startswith(b"Report tarball is at"):
+                    tarball_location = line.split(b"Report tarball is at ")[0].decode()
+                    break
```

Comment:
This fails, because the `line.startswith()` check is not correct (note the indentation in the command output).

Also, I think the index should be 1 instead of 0.

```bash
Building the report tarball
  Report tarball is at /var/snap/microk8s/2645/inspection-report-20211115_100636.tar.gz
```
```suggestion
            for line in output.split(b"\n"):
                line = line.decode().strip()
                if line.startswith("Report tarball is at"):
                    tarball_location = line.split("Report tarball is at ")[1]
                    break
```

Reply:
Example for the index:

```
In [3]: "Report tarball is at /var/location".split("Report tarball is at ")
Out[3]: ['', '/var/location']
```

Reply:
I have updated the code as suggested. Thanks!